runtime/compiler/env/J9ObjectModel.hpp

```diff
@@ -121,8 +122,14 @@ class ObjectModel : public OMR::ObjectModelConnector
    */
    MM_GCWriteBarrierType writeBarrierType() { return _writeBarrierType; }
 
+    /**
+   * @brief: Returns whether or not object references are compressed
+   */
```

Comment:
Indentation, also I don't think Doxygen will like the colon after the brief tag.

Reply:
I believe I copied this from a few lines above...

Reply:
Looks like I did get an extra space, but the colon is definitely there elsewhere.

Reply:
I'll fix all of the colons after testing completes.